bin/run-ci

```diff
@@ -23,6 +23,7 @@ sleep 5
 
 $DIR/waitfor-platform
 
+sleep 20
```

Comment:
I have added this as an attempted workaround for new, minor CI problem https://github.com/astronomer/issues/issues/1019